LICENSE

```diff
@@ -1,55 +1,213 @@
+      unless required by applicable law (such as deliberate and grossly
+      negligent acts) or agreed to in writing, shall any Contributor be
+      liable to You for damages, including any direct, indirect, special,
+      incidental, or consequential damages of any character arising as a
+      result of this License or out of the use or inability to use the
+      Work (including but not limited to damages for loss of goodwill,
+      work stoppage, computer failure or malfunction, or any and all
+      other commercial damages or losses), even if such Contributor
+      has been advised of the possibility of such damages.
+
+   9. Accepting Warranty or Additional Liability. While redistributing
+      the Work or Derivative Works thereof, You may choose to offer,
+      and charge a fee for, acceptance of support, warranty, indemnity,
+      or other liability obligations and/or rights consistent with this
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
+
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2012-present Apache Software Foundation
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+
+======================================================================
+Apache Myriad Subcomponents:
+
+The Apache Myriad project contains subcomponents with separate copyright
+notices and license terms. Your use of the source code for the these
+subcomponents is subject to the terms and conditions of the following
+licenses.
```

Comment:
I created a JIRA to discuss this. Short version, our NOTICE file had some
outdated dependencies listed.

https://issues.apache.org/jira/browse/MYRIAD-150

On Thu, Oct 15, 2015 at 12:52 PM, Adam <notifications@github.com> wrote:

> In LICENSE
> <https://github.com/apache/incubator-myriad/pull/7#discussion_r42149729>:
>
> > +       http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +   Unless required by applicable law or agreed to in writing, software
> > +   distributed under the License is distributed on an "AS IS" BASIS,
> > +   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +   See the License for the specific language governing permissions and
> > +   limitations under the License.
> > +
> > +
> > +======================================================================
> > +Apache Myriad Subcomponents:
> > +
> > +The Apache Myriad project contains subcomponents with separate copyright
> > +notices and license terms. Your use of the source code for the these
> > +subcomponents is subject to the terms and conditions of the following
> > +licenses.
>
> What subcomponents, and what following licenses? Are we adding these in a
> later commit?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-myriad/pull/7/files#r42149729>.
>